### PR TITLE
Resolve cs binaries from VirtusLab/coursier-m1 for aarch64

### PR DESCRIPTION
**Pull request by julienrf**

### CODE REVIEW

**Comment by julienrf** [comment]
Alternatively, would it be possible to use the VirtusLab runners directly from the coursier repo? I think that would simplify things by keeping them all in the same place, instead of spreading them over two repositories…

File: apps/resources/cs.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "name": "cs",
   "prebuiltBinaries": {
     "x86_64-pc-linux": "gz+https://github.com/coursier/coursier/releases/download/v${version}/cs-x86_64-pc-linux.gz",
-    "aarch64-pc-linux": "gz+https://github.com/coursier/coursier/releases/download/v${version}/cs-aarch64-pc-linux.gz",
+    "aarch64-pc-linux": "gz+https://github.com/VirtusLab/coursier-m1/releases/download/v${version}/cs-aarch64-pc-linux.gz",
```
Comment (alexarchambault) [inline apps/resources/cs.json:13]: That only applies to `2.1.0-RC5` and later versions. Maybe we should put the former URL in a [version override](https://github.com/coursier/apps/blob/fd91e1a23fea4abcdb4a1c490c5b0dd1e03f046d/apps/resources/scala.json#L12).

Comment (julienrf) [reply apps/resources/cs.json:13]: Yes, that’s a good point

File: apps/resources/cs.json [hunk 2]
```diff
@@ -10,12 +10,22 @@
...
     "x86_64-apple-darwin": "gz+https://github.com/coursier/coursier/releases/download/v${version}/cs-x86_64-apple-darwin.gz",
     "aarch64-apple-darwin": "gz+https://github.com/VirtusLab/coursier-m1/releases/download/v${version}/cs-aarch64-apple-darwin.gz",
     "x86_64-pc-win32": "zip+https://github.com/coursier/coursier/releases/download/v${version}/cs-x86_64-pc-win32.zip"
   },
   "versionOverrides": [
+    {
+      "versionRange": "]2.0.16, 2.1.0-RC4]",
```
Comment (alexarchambault) [inline apps/resources/cs.json:20]:
```suggestion
      "versionRange": "(2.0.16, 2.1.0-RC4]",
```

**Pull request merged**